garrysmod/resource/localization/en/main_menu.properties

```diff
@@ -470,3 +470,19 @@ ugc_upload.tag_scenes_save=Scene / Pose
 ugc_upload.tag_courses=Course
 ugc_upload.tag_posed=Pose / Ragdoll
 
+Option_Crosshair.Title=Default Crosshair Setup
+Option_Crosshair.Style_HL2=Half-Life 2
+Option_Crosshair.Style_Dot=Dot Image
+Option_Crosshair.Style_Classic=Classic
+Option_Crosshair.Gap=Gap
+Option_Crosshair.Size=Size
+Option_Crosshair.Thickness=Thickness
+Option_Crosshair.Dot=Dot
+Option_Crosshair.T_Style=T-Style
+Option_Crosshair.Alpha=Use Alpha
+Option_Crosshair.Quick_Info=Quick Info
+Option_Crosshair.Outline=Outline
+Option_Crosshair.Outline_Thickness=Outline Thickness
+Option_Crosshair.Preview_1=Preview 1
+Option_Crosshair.Preview_2=Preview 2
+Option_Crosshair.Reset_Crosshair=Reset Crosshair
```

Comment:
The localization strings (`Option_Crosshair.Reset_Crosshair` and others) must be all lowercase to match the rest of the localization strings.

Reply:
Done.